### PR TITLE
bumps ecto/basics to current version

### PR DESCRIPTION
**Pull request by gemantzu**
@zorbash / @Kosmas / @angelospap7 please review. Thanks.

### CODE REVIEW

File: gr/lessons/ecto/basics.md [hunk 1]
```diff
@@ -1,364 +1,233 @@
...
-  [applications: [:ecto, :postgrex]]
-end
-```
+### Ξεκινώντας
 
-### Αποθετήριο
+Στα πλαίσια αυτού του μαθήματος θα καλύψουμε τρία μέρη του Ecto:
 
-Τέλος χρειάζεται να δημιουργήσουμε το αποθετήριο του project μας, το κάλυμμα της βάσης δεδομένων.  Αυτό μπορεί να γίνει μέσω της εργασίς `mix ecto.gen.repo -r FriendsApp.Repo`.  Θα καλύψουμε τις εργασίες mix του Ecto στη συνέχεια.  Το αποθετήριο μπορεί να βρεθεί στο `lib/<όνομα του project>/repo.ex`:
+* Αποθετήριο - παρέχει τη διεπαφή στη βάση δεδομένων μας, συμπερλιλαμβανομένης της σύνδεσης
```
Comment (angelospap7) [inline gr/lessons/ecto/basics.md:26]: συμπεριλαμβανομένης

File: gr/lessons/ecto/basics.md [hunk 2]
```diff
@@ -1,364 +1,233 @@
...
-    where: u.confirmed == true,
-    select: count(u.id, :distinct)
-  )
-```
+Αν δείτε σφάλματα, βεβαιωθείτε ότι οι πληροφορίες ρύθμισης είναι σωστές και ότι η postgres διεργασία στο σύστημά σας τρέχει.
 
+### Μετατροπές
 
+Για να δημιουργήσετε και να αλλάξετε πίνακες στη βάση δεδομένων, το Ecto μας παρέχει τις μετατροπές.
+Κάθε μετατροπή περιγράφει ένα σύνολο δράσεων που θα γίνουν στη βάση δεδομένων μας, όπως ποιόν πίνακα να δημιουργήσει ή να αλλάξει.
```
Comment (angelospap7) [inline gr/lessons/ecto/basics.md:125]: ποιον χωρίς τόνο

File: gr/lessons/ecto/basics.md [hunk 3]
```diff
@@ -1,364 +1,233 @@
...
-end
+```elixir
+iex> %{person | age: 18}
+%Friends.Person{age: 18, name: "Tom"}
+iex> Map.put(person, :name, "Jerry")
+%Friends.Person{age: 18, name: "Jerry"}
 ```
 
-Αυτό ήταν!  Τώρα είστε έτοιμοι να αποθηκεύσετε μερικά δεδομένα.
+Στο επόμενο μάθημά μας για τα Σετ Αλλαγών, θα δούμε πως να επιβεβαιώσουμε τις αλλαγές δεδομένων μας και τελικά πως να τις οριστικοποιήσουμε στη βάση δεδομένων μας.
```
Comment (angelospap7) [inline gr/lessons/ecto/basics.md:233]: πώς με τόνο και στα δύο

**Review by angelospap7 — COMMENTED** [review]
It seems OK to me, but you have to fix some spelling mistakes that I commented.

File: gr/lessons/ecto/basics.md (hunk 3 above)
Comment (gemantzu) [reply gr/lessons/ecto/basics.md:233]: Μετά από ερώτηση σε εκπαιδευτικό, σε ενημερώνω πως μου είπε να μην βάλω τόνο εδώ...

**Comment by gemantzu** [comment]
Thanks for the spelling correction @angelospap7 ! The two first ones were resolved, the last one was dismissed as per an instructors directions :)

**Comment by gemantzu** [comment]
@angelospap7 / @Kosmas  can you please take another look at this? Thanks

**Review by Kosmas — APPROVED** [review]
@gemantzu looks fine to me :+1:

**Comment by gemantzu** [comment]
@elixirschool/developers can someone approve this so we can merge? I can't approve my own PR...

**Review by zorbash — APPROVED** [review]

**Review by doomspork — APPROVED** [review]

**Pull request merged**